src/Symfony/Bridge/PhpUnit/DeprecationErrorHandler/Deprecation.php

```diff
@@ -227,7 +230,18 @@ public function isMuted()
      */
     public function getType()
     {
-        if (self::PATH_TYPE_SELF === $pathType = $this->getPathType($this->triggeringFile)) {
+        $pathType = $this->getPathType($this->triggeringFile);
+        if ($this->languageDeprecation) {
+            switch ($pathType) {
+                case self::PATH_TYPE_VENDOR:
```

Comment:
Do we really need the switch here? Because of the following `if` statements in line 244 and 247 we only have to kandle the `PATH_TYPR_VENDOR` here. 
```
if ($this->languageDeprecation && self::PATH_TYPE_VENDOR === $pathType) {
    return self::TYPE_INDIRECT;
}
```
This would have the same behavior here or am I wrong?